### ISSUE TITLE
Android, buttons invalid drawing

### ISSUE THREAD

**Issue by bowfield** [kotleni/FloreProject#2]
**OS:** Android 8.0 (Xiaomi)
**Commentary:** Don't drawing text on any button.

![telegram-cloud-photo-size-2-5386643658343823841-y](https://user-images.githubusercontent.com/61633965/148613717-7c84cd70-4f14-4491-b3c1-6ed7b7b5d93a.jpg)

**Comment by kotleni** [comment]
Yeah, its normal. Android have not playable support now. _Fix it in next updates._
